src/main/drivers/bus_i2c_impl.h

```diff
@@ -26,13 +26,20 @@
 #define I2C_LONG_TIMEOUT             ((uint32_t)(10 * I2C_SHORT_TIMEOUT))
 #define I2C_DEFAULT_TIMEOUT          I2C_SHORT_TIMEOUT
 
-#define I2C_PIN_SEL_MAX 3
+#define I2C_PIN_SEL_MAX 4
+
+typedef struct i2cPinDef_s {
```

Comment:
Braces around initializer are necessary for non-F4 targets too

Reply:
Very true...

Reply:
`typedef ioTag_t i2cPinDef_t;` is one possibility, but it may be too confusing ...